e2e_tests_codefresh_pipeline.yaml

```diff
@@ -0,0 +1,71 @@
+version: "1.0"
+stages:
+  - "clone"
+  - "build"
+  - "e2e_test"
+steps:
+  main_clone:
+    title: "Cloning repository"
+    type: "git-clone"
+    repo: "Labelbox/labelbox-python"
+    stage: "clone"
+    revision: "${{CF_BRANCH}}"
+  parallel_build:
+    type: "parallel"
+    stage: "build"
+    steps:
+      build_sdk_test:
+        description: "Building Docker image for sdk tests"
+        tag: "testing"
+        type: "build"
+        image_name: "sdk-tests"
+        disable_push: true
+        working_directory: "${{main_clone}}"
+        dockerfile: "Dockerfile"
+  e2e_test:
+    title: "Python SDK e2e tests"
+    description: "Running Python SDK e2e tests"
+    fail_fast: false
+    stage: "e2e_test"
+    image: "${{build_sdk_test}}"
+    working_directory: "IMAGE_WORK_DIR"
+    commands:
+      - pytest
+  sendSlackSuccessToPagesActiveLearning:
+    title: "Alert on success to pages active learning"
+    type: slack-message-sender
+    arguments:
+      WEBHOOK_URL: "${{slack_webhook_url}}"
+      MESSAGE: "SDK Tests have succeeded-- ${{CF_BUILD_URL}}"
+    when:
+      condition:
+        all:
+          succeeded: e2e_test.result == 'success'
+  sendSlackFailureToE2E:
```

Comment:
I'm not super familiar with codefresh pipelines. I assume that these names are just arbitrary stage names. Is there a reason some are camel case and some are snake case?